partners.yml

```diff
@@ -748,10 +749,14 @@ components:
           type: number
           description: One time fee to be paid at the beginning of the loan in SEK
           minimum: 0
-        monthlyFee:
+        administrationFee:
           type: number
           description: Fee to be paid every month until loan fully re-paid in SEK
           minimum: 0
+        invoiceFee:
+          type: number
+          description: Fee to be paid every month until loan fully re-paid in SEK if not connected to autogiro
```

Comment:
I'd rephrase it more something like `Fee that will be charge for each invoice if customer did not allow direct debit from his bank account`